### PR TITLE
Update README.md with Windows/macOS/Linux instructions

### PR DESCRIPTION
**Pull request by lasers**
I add Linux instructions then splits up `README.md` into separate platforms.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,33 +2,67 @@
...
-3. Submit a pull request
+### Restore
+1. Download the [repo](https://github.com/Florin9doi/WiiBanners/zipball/master)
+2. Copy the files to your Wii NAND Root.
+
+## macOS
+
+### Submit Banners with git
+```shell
+# git clone https://github.com/Florin9doi/WiiBanners.git ~/WiiBanners
```
Comment (Florin9doi) [inline README.md]: I prefer a numbered list, not a big code block with comments.

File: README.md [hunk 2]
```diff
@@ -2,33 +2,67 @@
...
+### Submit Banners with zip
+```shell
+cd ~/.local/share/dolphin-emu/
+find Wii/title -name 'banner.bin' | zip /tmp/WiiBanners-$(date +%s).zip -@
+# Open a new issue and attach `/tmp/WiiBanners-*.zip` archive.
+```
+
+### Restore with rsync
+```shell
+# git clone https://github.com/Florin9doi/WiiBanners.git ~/WiiBanners
```
Comment (Florin9doi) [inline README.md]: git isn't installed by default. Downloading the archive and dragging and dropping the files is preferred. Restore section may remain common.

**Comment by lasers** [comment]
Please review again. You can see what it looks like at https://github.com/lasers/WiiBanners/tree/linux.

**Comment by Florin9doi** [comment]
Thank you

**Pull request merged**